pkg/operation/botanist/monitoring.go

```diff
@@ -71,6 +70,26 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 		return err
 	}
 
+	var (
+		defaultSecretName  = "prometheus-tls"
```

Comment:
Maybe expose this as a constant somewhere and reuse it here and in `pkg/operation/botanist/secrets.go` where it is generated?